src/solution.c

```diff
@@ -396,6 +401,53 @@ static void sol_thd_sleep(systime_t *deadline, systime_t interval)
   chSysUnlock();
 }
 
+/** Calculates Iono and Tropo correction
+ * \param n_ready_tdcp The number of measurements written to `m_tdcp`
+ * \param nav_meas_tdcp Pointer to measurements array */
+void calc_iono_tropo(u8 n_ready_tdcp,
```

Comment:
should we hold off on these corrections for RTK for now? we might want to formalize our pre processing steps to make sure we're transmitting/logging raw pseudoranges but using corrected ones in the filter. with a short baseline they won't have much of an effect anyway